pkg/project/project.go

```diff
@@ -322,6 +322,11 @@ func DetectWithRevControl(
 }
 
 func obfuscateProjectName(ctx context.Context, folder string) string {
+	// when folder unknown, use Unknown Project (https://github.com/wakatime/wakatime-cli/issues/1164)
```

Comment:
let's create a test for this?

Reply:
Added with a39305f12c9309887780f38cddbebc8210856256.